packages/experiments/src/index.ts

```diff
@@ -1,5 +1,6 @@
 export * from './Chiclet';
 export * from './CollapsibleSection';
+export * from './Foundation';
```

Comment:
This is because of a nasty workaround of creating a component that Esteban had to do to enact styling which my PR #5693 actually resolves. That said, I'd rather recreate (and later delete) Foundation in the website package rather than have this export here.

Reply:
Scratch that idea. Instead, just use `createComponent` directly from Foundation. It should just require providing 2 additional types `ITheme` and `IProcessedStyleSet<ICollapsibleSectionTitleStyles>`.

Reply:
(Also, this export still should be removed.)

Reply:
Now that #5693 is merged changes to this file should be reverted entirely